src/_includes/macros/sub-nav.html

```diff
@@ -50,10 +50,10 @@
                 </ul>
             {%- endfor %}
             </div>
-        </section>
-        <aside class="sub-nav_media">
+        </div>
+        <div class="sub-nav_media">
```

Comment:
I think this one was appropriate as an aside. When navigating w/ VO it made it clear that it was extra content and not a typical nav item.